test/tap/groups/ai/pre-cleanup.bash

```
@@ -0,0 +1,54 @@
+# Check required environment variables
+if [ -z "${INFRA_ID:-}" ]; then
+    echo "WARNING: INFRA_ID is not set, skipping MCP cleanup"
+    exit 0
+fi
+
+if [ -z "${WORKSPACE:-}" ]; then
+    echo "WARNING: WORKSPACE is not set, skipping MCP cleanup"
+    exit 0
+fi
+
+export ROOT_PASSWORD=$(echo -n "${INFRA_ID}" | sha256sum | head -c 10)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)